### PR TITLE
Redis.configuration.connection & Sidekiq.configuration.queues_size

### LINKED ISSUE #28: Redis client is nil
My configuration as:
```ruby
HealthMonitor.configure do |config|
  config.database
  config.cache
  config.sidekiq
  config.sidekiq.configure do |sidekiq_config|
    sidekiq_config.latency = 1.minute
  end

  config.redis
  if ENV['REDIS_URL']
    config.redis.configure do |redis_config|
      redis_config.url = ENV['REDIS_URL']
    end
  end
end
```

Brings redis error:
`undefined method `client' for nil:NilClass`

In development environment i use local redis, but for staging and production I extract redis url from ENV.

### PR DESCRIPTION
**Pull request by sveredyuk**
Closed #28

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,7 +151,8 @@ Some of the providers can also accept additional configuration:
 # Sidekiq
 HealthMonitor.configure do |config|
   config.sidekiq.configure do |sidekiq_config|
-    sidekiq_config.latency = 3.hours
+    sidekiq_config.latency    = 3.hours
```
Comment (lbeder) [inline README.md]: Could you align the `=`?

File: README.md [hunk 2]
```diff
@@ -171,6 +172,7 @@ The currently supported settings are:
 #### Sidekiq
 
 * `latency`: the latency (in seconds) of a queue (now - when the oldest job was enqueued) which is considered unhealthy (the default is 30 seconds, but larger processing queue should have a larger latency value).
+* `queue_size`: the size (maximim) of a queue which is considered unhealthy (the default is 100.
```
Comment (lbeder) [inline README.md]: Could you add the missing `)`?

Comment (sveredyuk) [reply README.md]: Ofc.

File: README.md (hunk 1 above)
Comment (sveredyuk) [reply README.md]: Yes

**Pull request merged**